packages/scaffold-utils/src/SolanaStoreUtil.ts

```diff
@@ -0,0 +1,122 @@
+import { proxy, ref, subscribe as sub } from 'valtio/vanilla'
+import { subscribeKey as subKey } from 'valtio/utils'
+import { Connection } from '@solana/web3.js'
+
+import UniversalProvider from '@walletconnect/universal-provider'
+
+import type { Chain, CombinedProvider, Provider } from './SolanaTypesUtil.js'
+
+type StateKey = keyof SolStoreUtilState
+
+export interface SolStoreUtilState {
+  projectId: string
```

Comment:
is this required? projectId is available in `OptionsController`

Reply:
Yes, moved it out and using OptionsController now